src/App.js

```diff
@@ -9,28 +9,54 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData,
-      view: 'mainPage'
+      // movieData: movieData,
+      movieData: {
+        movies: []
+      },
+      view: 'mainPage', 
+      currentMovieId: 0,
+      error: '',
     }
   }
 
-goToMain = () => {
-  this.setState({view: 'mainPage'})
-}
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
If you haven't already (sorry for the lateness of this PR), be sure to move your fetches into their own file.